cluster-provision/k8s/1.18/manifests/cnao/network-addons-config.crd.yaml

```diff
@@ -26,6 +26,7 @@ spec:
           type: object
         status:
           type: object
+      type: object
```

Comment:
0_o ... why is this suddenly needed ?

Reply:
No clue, I downloaded from CNAO release pagehttps://github.com/kubevirt/cluster-network-addons-operator/releases/tag/0.39.2

Reply:
It became a requirement for CRDs in recent versions. Previously, the top level object was treated as `object` implicitly, but it was added that it has to be explicit.